analyzer/src/funTest/assets/projects/synthetic/pip-expected-output.yml

```diff
@@ -34,24 +34,24 @@ packages:
 - id: "PyPI::click:6.7"
   purl: "pkg:pypi/click@6.7"
   authors:
-  - "Armin Ronacher"
+  - "Armin Ronacher <armin.ronacher@active-4.com>"
   declared_licenses:
-  - "BSD License"
+  - ""
```

Comment:
We should never store an empty license. This needs to be fixed in the implementation.